dotnet/dotNetResponse.cs

```diff
@@ -1,5 +1,5 @@
 /*
```

Comment:
needs additional review from someone with .NET skills. Should there also be changes to the request side too?